packaging/gieros2021/data_assembly.py

```diff
@@ -0,0 +1,26 @@
+from brainio.assemblies import BehavioralAssembly
+from brainio.packaging import package_data_assembly
+
+assembly = BehavioralAssembly(['dog', 'dog', 'cat', 'dog', ...],
+                               coords={
+                                   'image_id': ('presentation', ['image1', 'image2', 'image3', 'image4', ...]),
+                                   'sample_object': ('presentation', ['dog', 'cat', 'cat', 'dog', ...]),
+                                   'distractor_object': ('presentation', ['cat', 'dog', 'dog', 'cat', ...]),
+                                   # ...more meta
+                                   # Note that meta from the StimulusSet will automatically be merged into the
+                                   #  presentation dimension:
+                                   #  https://github.com/brain-score/brainio/blob/d0ac841779fb47fa7b8bdad3341b68357c8031d9/brainio/fetch.py#L125-L132
+                               },
+                               dims=['presentation'])
+assembly.name = '<authoryear>'  # give the assembly an identifier name
+
+# make sure the assembly is what you would expect
+assert len(assembly['presentation']) == 179660
+assert len(set(assembly['image_id'].values)) == 1600
+assert len(set(assembly['choice'].values)) == len(set(assembly['sample_object'].values)) \
+       == len(set(assembly['distractor_object'].values)) == 2
+
+# upload to S3
+# package_data_assembly(assembly, assembly_identifier=assembly.name, ,
```

Comment:
Do you have the latest `main` of BrainIO?  The packaging methods now require the name of the catalog you want to add to.  In this case it will be `brainio_brainscore`.  You can find the name in setup.py:  https://github.com/brain-score/brain-score/blob/919708b87e615ca7c0b7a7c7d088b3d389ddb928/setup.py#L62